src/commands/install-packages.yml

```diff
@@ -1,7 +1,8 @@
 description: >
   Setup a python environment and install the packages for your project either globally with pip or in a virtualenv with poetry or pipenv.
   With pip as pkg-manager, the command will assume `-r requirements.txt`.
-  For poetry and pipenv, no args are provided. Expect the default caching locations for packages and virtualenvs on a debian system with pyenv.
+  With poetry as pkg-manager, the command will assome `--no-ansi`.
```

Comment:
Minor typo here.

Reply:
Thank you for spotting this, it's fixed now. :slightly_smiling_face: